end2end/tests/adminPanelWorkflowEditor.spec.ts

```diff
@@ -10,98 +16,160 @@ test('Create a new workflow and add a step and an action', async ({ page }) => {
+                await page.getByRole('button', { name: 'Yes' }).click();
+                await page.getByRole('button', { name: 'Delete Workflow' }).click();
+                await page.getByRole('button', { name: 'Yes' }).click();
+            }
+        }    
+    }
 });
 
-test('View workflow history', async ({ page }) => {
-    // Generate unique workflow title
-    const testID = getRandomId();
-    const workflowTitle = `New_Workflow_${testID}`;
-
-    await page.goto(LEAF_URLS.WORKFLOW_EDITOR);
-
-    await page.locator('#btn_newWorkflow').click();
-
-    // Wait for the "Create new workflow" dialog to be visible
-    const workflowCreateDialog = page.locator('span.ui-dialog-title:has-text("Create new workflow")');
-    await expect(workflowCreateDialog).toBeVisible();
-
-    await page.locator('#description').fill(workflowTitle);
-
-    const saveButton = page.locator('#button_save');
-    await saveButton.click();
-
-    // Assert that the newly created workflow is visible
-    await expect(page.locator('a').filter({ hasText: workflowTitle })).toBeVisible();
-
-    await page.locator('#btn_viewHistory').click();
-
-    // Wait for the Workflow History dialog to become visible
-    const workflowHistoryDialog = page.locator('span.ui-dialog-title:has-text("Workflow History")');
-    await expect(workflowHistoryDialog).toBeVisible();
-
-    // Verify if the new workflow name appears in the history
-    await expect(page.locator('#historyName')).toContainText(workflowTitle);
+test('Changing Form Field appears in Workflow History', async ({ page }) => {
+
+    let formID = '';
+    let [ workflowID, stepID ] = [ '', ''];
+    
+    const testId = getRandomId();
+    const workflowName = `workflow_name_${testId}`;
+    const stepTitle = "Step 1";
+    const stepElement = await page.getByLabel(`workflow step: ${stepTitle}`, { exact: true });
+
+    try {
+
+        // Create workflow with one step
+         [ workflowID, stepID ] = await createBaseTestWorkflow(page, workflowName, stepTitle);
+        await expect(stepElement).toBeVisible();
+
+        // Create form and add a section with a Single line text
+        formID = await createTestForm(page, `form_name_${testId}`, `form_descr_${testId}`);
+        const textSectionID = await addFormQuestion(page, 'Add Section', 'Single line text', 'Single line text');
+        await page.getByLabel('Workflow: No Workflow. Users').selectOption(workflowID);
+
+        // Go back to the workflow
+        await loadWorkflow(page, workflowID);
+        await expect(stepElement).toBeVisible();
+
+        // Set the Single Line Text form field on the step
+        await stepElement.click();
+        await page.getByLabel('Form Field:').selectOption(textSectionID);
+        await stepElement.click();
+
+        // View History and verify that the form field change is listed
+        await page.getByRole('button', { name: 'View History' }).click();
```

Comment:
Could you add this line to keep confirmation of the title?  TY.

await expect(page.locator('#historyName')).toContainText(workflowName);